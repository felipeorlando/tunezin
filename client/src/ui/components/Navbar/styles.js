import styled from 'styled-components'
import Container from '../Container'

export const StyledContainer = styled(Container)`
  justify-content: center;
  display: flex;
`

export const StyledLogo = styled.h1`
  color: ${({ theme }) => theme.dark};
  font-size: 1rem;
`

export const StyledWrapper = styled.nav`
  padding: 15px 0;
`
