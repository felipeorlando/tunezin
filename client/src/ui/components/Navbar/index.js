import { StyledContainer, StyledLogo, StyledWrapper } from './styles'

const Navbar = () => (
  <StyledWrapper>
    <StyledContainer>
      <StyledLogo>tunezin</StyledLogo>
    </StyledContainer>
  </StyledWrapper>
)

export default Navbar
