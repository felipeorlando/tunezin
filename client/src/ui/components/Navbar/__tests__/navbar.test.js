import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import Navbar from '..'

describe('Navbar', () => {
  it('renders correctly', () => {
    const component = <Navbar />
    const tree = renderer.create(component).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders with logo', () => {
    render(<Navbar />)
    const logoElement = screen.getByText(/tunezin/i)
    expect(logoElement).toBeInTheDocument()
  })
})
