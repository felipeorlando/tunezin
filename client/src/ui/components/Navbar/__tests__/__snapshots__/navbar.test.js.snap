// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<nav
  className="sc-hKgILt ikNWTJ"
>
  <div
    className="sc-bdfBwQ sc-gsTCUz djeIjE czCETW"
  >
    <h1
      className="sc-dlfnbm bQQYnX"
    >
      tunezin
    </h1>
  </div>
</nav>
`;
